### PR TITLE
Handle repomd.xml that don't contain size element

### PR DESCRIPTION
**Pull request by srinidhibs**
Parse `repomd.xml` files that do not contain the `<size>` element.

### CODE REVIEW

File: lib/repomd_parser/repomd_xml_parser.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def parse
         checksum_type: hash[:checksum][:type],
         checksum: hash[:checksum][:value],
         type: type,
-        size: hash[:size][:value].to_i
+        size: hash[:size] ? hash[:size][:value].to_i : 0
```
Comment (ikapelyukhin) [inline lib/repomd_parser/repomd_xml_parser.rb]: I've though about this for a while, perhaps `nil` would be better than 0:
* 0 size indicates that the file is empty (in theory, a repo can contain empty files);
* `nil` indicates that the size attribute is missing, which is the case with old-style repos.

What do you think, @srinidhibs?

**Review by ikapelyukhin — APPROVED** [review]
Other than 0/`nil` thoughts, looks good!

File: lib/repomd_parser/repomd_xml_parser.rb (hunk 1 above)
Comment (srinidhibs) [reply lib/repomd_parser/repomd_xml_parser.rb]: Sounds good @ikapelyukhin! Will push the changes in a bit.

**Comment by ikapelyukhin** [comment]
Let's merge this! :+1:

**Comment by ikapelyukhin** [comment]
Released: https://rubygems.org/gems/repomd_parser/versions/0.1.2

**Comment by srinidhibs** [comment]
Thank you! 🙂

**Pull request merged**